templates/news-detail/news-detail.css

```diff
@@ -0,0 +1,99 @@
+.news-detail {
+  main {
+    position: relative;
+    padding-top: 12.5rem;
+
+    .hero {
+      position: absolute;
+      top: 0;
+      height: 33.75rem;
+      width: 100%;
+      margin: 0;
+      z-index: -1;
+
+      picture {
+        object-fit: cover;
+
+        img {
+          width: 100%;
+          height: 100%;
+          object-fit: cover;
+        }
+      }
+    }
+
+    .section {
+      margin: 0;
+
+      .default-content-wrapper {
+        max-width: 76.4375rem;
+        width: calc(100% - 80px);
+        margin: 0 auto;
+        border-radius: 4px;
+        background: #fff;
+        box-shadow: 4px 20px 40px 4px rgba(0 0 0 / 10%);
+        padding: 2.2rem 6.5625rem 12.5rem;
+
+        p.centered { text-align: center }
```

Comment:
currently font size is being fetched from styles.css
As per current website, lets have it as 1 rem across all viewport.

Reply:
let's also adjust padding/margin in mobile view.